src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${
+        this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${
+        this.coords.y}`;
+    },
+
+    goForward(step = 1) {
+      if (step >= 0) {
```

Comment:
Zero wouldn't make any sense here, check it throughout your code
```suggestion
      if (step > 0) {
```